plugins/kubernetes/test/models/kubernetes/template_filler_test.rb

```diff
@@ -786,6 +786,29 @@ def secret_annotations(hash)
       end
     end
 
+    describe "podtemplate" do
+      let(:result) { template.to_hash }
+      let(:containers) { result.dig_fetch(:template, :spec, :containers) }
+      let(:container) { containers.first }
+      let(:build) { builds(:docker_build) }
+      let(:image) { build.docker_repo_digest }
+
+      before do
+        raw_template.replace \
+          YAML.safe_load(
+            read_kubernetes_sample_file('kubernetes_podtemplate.yml')
+          ).deep_symbolize_keys
```

Comment:
FYI same but less funky :D
```suggestion
        raw_template.replace(
          YAML.safe_load(
            read_kubernetes_sample_file('kubernetes_podtemplate.yml')
          ).deep_symbolize_keys
        )
```

Reply:
thats only a matter of opinion ;)

no worries, have amended. Thanks!